process-automation/latest/model/authentication.md

```diff
@@ -13,11 +13,11 @@ The basic properties of creating authentication are:
 | Authentication name | *Required.* The name used to identify the authentication. The name must be in lowercase and be between one and 26 characters in length, for example `token1auth`. |
 | Authentication description | *Optional.* A description of the authentication. This can be your identity provider, for example `Keycloak authentication used`. |
 | Authentication type | *Required.* Select the type of authentication, for example `Basic`. There are three types of authentication you can use, `Basic`, `Client credentials`, and `Bearer token`. |
-| Secured | *Optional.* When selected, credentials are defined during deployment time, or de-select the checkbox to provide them during modeling time. These credentials are used when contacting the external API. Note: All the configuration parameters can be overridden at deployment time. |
```

Comment:
Are you sure that this sentence is not valid anymore?
_Note: All the configuration parameters can be overridden at deployment time._

![image](https://user-images.githubusercontent.com/101395584/168537288-b22d1f41-2878-47dd-b46b-fcb30663b863.png)

Reply:
Morning @Artur-Piechuta 

Yep there is a Note at the bottom of those fields.
![image](https://user-images.githubusercontent.com/35293859/168548749-1243932a-1fbd-479c-894e-22f74ffca153.png)